pages/htmlresources.md

```diff
@@ -13,7 +13,7 @@ Documentation helps to understand overall progress of project. Visit following l
 
 Prerequisites:
 
-* [Couchapp](#couchapp-installation)
+* [Couchapp](htmlresources.md#Couchapp_Installation)
```

Comment:
@praneetharra I think you branch off of one of your feature branches
that is why you have this file too much ...